File: php/chutenumero.php
<?php
$numero = 5;
$randomnumero = rand(0, 10);
if($numero == $randomnumero) {
    print('Acertou!');
} else {
    if($numero > $randomnumero) {
        print('Chutou alto!');
    } else {
        print('Chutou baixo!');
    }
}

?>

File: rust/chutenumero.rs
use rand::Rng;

fn main() {
    let numero = 5;
    let mut rng = rand::thread_rng();
    let randomnumero = rng.gen_range(0..10);
    if numero == randomnumero {
        return println!("Acertou!");
    }
    if numero > randomnumero {
        println!("Chutou Alto!");
    } else {
        println!("Chutou Baixo!");
    }
}